packages/data-stores/src/plans/selectors.ts

```diff
@@ -54,10 +80,34 @@ export const getPlansPaths = ( state: State ): string[] => {
 
 export const getPrices = ( state: State, _: string ): PricesMap => state.prices;
 
+export const getDiscounts = ( state: State, locale: string ): DiscountsMap => {
+	// call getPrices to trigger the resolver in case someone only wants to get the discounts
+	select( STORE_KEY ).getPrices( locale );
```

Comment:
This call being async I assume [discounts would be 0 initially](https://github.com/Automattic/wp-calypso/pull/48790/files#diff-7e3bb074454914d1e91ffecc3680b670e78258206e741a330ec47befde62356eR45) instead of `undefined` which may cause some jumping in the UI.

Reply:
That's a good point — does this consideration apply also to prices, which [are set to an empty string](https://github.com/Automattic/wp-calypso/pull/48790/files#diff-7e3bb074454914d1e91ffecc3680b670e78258206e741a330ec47befde62356eR33) by default instead of `undefined`?

Reply:
Yes, but when price is empty string I believe we still show placeholders. Discounts might be 0 sometime.